scripts/rebuild.sh

```diff
@@ -14,4 +14,5 @@ npm ci
 rm -rf ../en
 node generatePages.js
 node generateDocs.js ../vcpkg/docs
+node generatePackages.js ../vcpkg
```

Comment:
How would the token be used?

Reply:
A PAT can be provided as the second argument. We can make `rebuild.sh` pass the value of an environment variable.